tests/schemas/rules/value/concatenate.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "description": "Concatenate two answers space seperated",
+    "rule": {
+      "concatenate": [
+        [
+          {
+            "source": "answers",
+            "identifier": "first-name"
+          },
+          {
+            "source": "answers",
+            "identifier": "last-name"
+          }
+        ],
+        " "
+      ]
+    }
+  },
+  {
+    "description": "Concatenate two answers comma seperated",
+    "rule": {
+      "concatenate": [
+        [
+          {
+            "source": "answers",
+            "identifier": "first-name"
+          },
+          {
+            "source": "answers",
+            "identifier": "last-name"
+          }
+        ],
+        ","
+      ]
+    }
+  },
+  {
+    "description": "Concatenate two metadata",
```

Comment:
minor, but the other 2 you highlighted the separator.

Reply:
It is now one for answers with a space and one for metadata with comma